### PR TITLE
MANTA-5463 muntar has outlived its usefulness MANTA-5464 Want msync (manta-sync) #395: Want npm lock file

### PR DESCRIPTION
**Pull request by bahamat**
cc @bahamas10

### CODE REVIEW

**Comment by bahamat** [comment]
We may need/want stronger deprecation language around `muntar`.

File: docs/man/msync.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+muntar 1 "May 2023" Manta "Manta Commands"
```
Comment (travispaul) [inline docs/man/msync.md:1]: Looks like there are still a few references to muntar in this file.

**Comment by bahamas10** [comment]
yes!! 🎉 🎉 🎉

File: docs/man/msync.md (hunk 1 above)
Comment (bahamat) [reply docs/man/msync.md]: Thanks. Fixed.

**Comment by travispaul** [comment]
> We may need/want stronger deprecation language around `muntar`.

How do you feel about adding a `console.warn` with a note about it's deprecation when `bin/muntar` is invoked?

**Review by travispaul — DISMISSED** [review]

**Review by danmcd — COMMENTED** [review]
1.) Travis's deprecation-warning message question needs an answer.

2.) We need testing notes in the ticket(s). This can include msync's documented history of battle-tested success.

**Comment by bahamat** [comment]
Added deprecation notice to `muntar -h`.

Testing notes added to Jira.

**Review by travispaul — APPROVED** [review]

**Review by danmcd — APPROVED** [review]
I have not reviewed the existing msync which is now being imported.

I'd `integration-approval` this based on the testing, however.

**Pull request merged**